common/travis/changed_templates.sh

```diff
@@ -2,5 +2,11 @@
 #
 # changed_templates.sh
 
+if command -v chroot-git >/dev/null 2>&1; then
+	GIT_CMD=$(command -v chroot-git)
+elif command -v git >/dev/null 2>&1; then
```

Comment:
No need for -v if just routing to /dev/null?

Reply:
Without -v it'll try to execute `git`.

Reply:
-v searches for a command

![image](https://user-images.githubusercontent.com/30738253/53934095-168f6a80-4080-11e9-8360-9afbd279dd3d.png)

Reply:
Ah, did not realize